src/views/conference/2018/schedule/schedule.jsx

```diff
@@ -0,0 +1,186 @@
+const bindAll = require('lodash.bindall');
+const classNames = require('classnames');
+const connect = require('react-redux').connect;
+const PropTypes = require('prop-types');
+const React = require('react');
+
+const scheduleActions = require('../../../../redux/conference-schedule.js');
+
+const FlexRow = require('../../../../components/flex-row/flex-row.jsx');
+const SubNavigation = require('../../../../components/subnavigation/subnavigation.jsx');
+const TitleBanner = require('../../../../components/title-banner/title-banner.jsx');
+
+const render = require('../../../../lib/render.jsx');
+const Page = require('../../../../components/page/conference/2018/page.jsx');
+
+
+require('./schedule.scss');
+
+class ConferenceSchedule extends React.Component {
+    constructor (props) {
+        super(props);
+        bindAll(this, [
+            'handleScheduleChange',
+            'renderChunkItems'
+        ]);
+    }
+    componentDidMount () {
+        const day = window.location.hash.substr(1) || 'thursday';
+        this.handleScheduleChange(day);
+    }
+    handleScheduleChange (day) {
+        window.history.replaceState(history.state, '', `#${day}`);
```

Comment:
This is probably copied from previous years, but kind of odd we're using `replaceState` with hashes when we could be using real URLs.